### PR TITLE
Update FastPitch

### PR DESCRIPTION
**Pull request by blisc**

### CODE REVIEW

File: examples/tts/conf/fastpitch_align.yaml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+name: "FastPitch"
+sample_rate: 44100
```
Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:2]: All our released models in TTS uses `sample_rate=22050`. Do you plan to release something with 44100?

Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:2]: All our released models in TTS use sample_rate=22050. Do you plan to release something with 44100? I would suggest to convert all values related to audio params according to sample_rate=22050.

File: examples/tts/conf/fastpitch_align.yaml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+      normalize: true
+      sample_rate: ${sample_rate}
+      trim: false
+      sup_data_path: ${prior_folder}
+      n_window_stride: ${model.n_window_stride}
+      n_window_size: ${model.n_window_size}
+      pitch_fmin: 80
+      pitch_fmax: 640
+      pitch_avg: 216.23100173462166
+      pitch_std: 46.24582950436719
```
Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:66]: I would suggest to convert all values related to audio params according to LJ Speech.

File: examples/tts/conf/fastpitch_align.yaml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
```
Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:33]: I would suggest to convert all values related to audio params according to LJ Speech.

File: examples/tts/conf/fastpitch_align.yaml [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+model:
+  learn_alignment: true
+  n_speakers: 1
+  symbols_embedding_dim: 384
+  max_token_duration: 75
+  n_mel_channels: 80
+  pitch_embedding_kernel_size: 3
+  n_window_size: 2048
+  n_window_stride: 512
```
Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:15]: I would suggest to convert all values related to audio params according to sample_rate=22050.

File: examples/tts/conf/fastpitch_align.yaml [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    d_model: ${model.symbols_embedding_dim}
+    d_head: 64
+    d_inner: 1536
+    kernel_size: 3
+    dropout: 0.1
+    dropatt: 0.1
+    dropemb: 0.0
+    # n_embed: 148  # NOTE Should match # of tokens in `symbol_set`
+    d_embed: ${model.symbols_embedding_dim}
+    # padding_idx: 0
```
Comment (Oktai15) [inline examples/tts/conf/fastpitch_align.yaml:118]: Let's remove commented lines and leave a comment that `padding_idx` and `n_embed` will be set in `__init__` of model

**Review by Oktai15 — CHANGES_REQUESTED** [review]
Great job! I left some minor comments

File: examples/tts/conf/fastpitch_align.yaml (hunk 1 above)
Comment (blisc) [reply examples/tts/conf/fastpitch_align.yaml]: done

File: examples/tts/conf/fastpitch_align.yaml (hunk 2 above)
Comment (blisc) [reply examples/tts/conf/fastpitch_align.yaml]: done

File: examples/tts/conf/fastpitch_align.yaml (hunk 3 above)
Comment (blisc) [reply examples/tts/conf/fastpitch_align.yaml]: done

File: examples/tts/conf/fastpitch_align.yaml (hunk 4 above)
Comment (blisc) [reply examples/tts/conf/fastpitch_align.yaml]: done

File: examples/tts/conf/fastpitch_align.yaml (hunk 5 above)
Comment (blisc) [reply examples/tts/conf/fastpitch_align.yaml]: done

**Review by Oktai15 — APPROVED** [review]

**Pull request merged**